.github/workflows/ci.yml

```diff
@@ -195,6 +195,16 @@ jobs:
           miniforge-variant: Mambaforge
           use-mamba: true
         if: ${{ ! contains(matrix.OS_NAME, 'Linux') }}
+      
+      - name: Patch license for Mambaforge deprecation
+        if: matrix.MINIFORGE_NAME == 'Mambaforge'
+        run: |
+          echo "!!!!!! Mambaforge is now deprecated !!!!!" > MAMBAFORGE_LICENSE.txt
+          echo "Future Miniforge releases will NOT build Mambaforge installers." >> MAMBAFORGE_LICENSE.txt
+          echo "We advise you switch to Miniforge at your earliest convenience." >> MAMBAFORGE_LICENSE.txt
```

Comment:
oh man, Miniforge vs Miniforge3..... naming decisions that come back to haunt us!!!!

Reply:
Yea... And how it's Miniforge3 for CPython but just Miniforge for PyPy :P

Reply:
Correct!!!